### PR TITLE
Feat take action page

### PR DESCRIPTION
**Pull request by busratok**
### Description

- All component on the takeAction page is created and added to the page
- DonationForm is embedded
- Link is added to the landing page to access the takeaction page

### CODE REVIEW

File: src/components/takeAction/inspireTomorrow.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+              professionals. Your expertise can transform lives, closing the
+              skills gap and empowering students for a brighter future. Join us
+              in shaping the tech leaders of tomorrow!
+            </p>
+            <Link
+              href="https://tally.so/r/mVPeyN"
+              className="px-4 py-2 text-sm font-semibold font-display rounded-lg border-2 border-cyan hover:text-gray-900 focus:text-gray-900 hover:bg-gray-200 focus:bg-gray-200 focus:outline-none focus:shadow-outline"
+            >
+              volunteer
+            </Link>
```
Comment (lucaCambi77) [inline src/components/takeAction/inspireTomorrow.tsx:45]: I see it respects the design but `volunteer` and `Get started` in the next section are leading to the same form. So maybe we should name them in the same way?

Comment (busratok) [reply src/components/takeAction/inspireTomorrow.tsx:45]: It makes sense. I will ask Stefan about it, if he approves, I'll go ahead and make the necessary changes."

File: src/components/takeAction/volunteering.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+              their professional experience. Make our mission your own!
+            </p>
+            <p className=" pt-9">
+              Someone who volunteers may not get paid, but they get so much
+              more.
+            </p>
+            <p className="pt-9">
+              Someone who volunteers may not get paid, but they get so much
+              more.
+            </p>
```
[the remark this answers is not in the document]
Comment (busratok) [reply src/components/takeAction/volunteering.tsx]: > Looks like this is repeating?

Fixed

File: src/components/takeAction/inspireTomorrow.tsx (hunk 1 above)
Comment (busratok) [reply src/components/takeAction/inspireTomorrow.tsx:47]: > It makes sense. I will ask Stefan about it, if he approves, I'll go ahead and make the necessary changes."

I asked Stefan and he said to go with volunteer. Change is made

**Review by lucaCambi77 — APPROVED** [review]

**Pull request merged**